### ISSUE TITLE
Notebook 04-arrays-operations

### ISSUE THREAD

**Issue by KevinMurphy806** [dpploy/chen-3170#17]
This notebook is a very god description of vectors and an intro to matrices.  The only thing I would add to this notebook is in the matrix section, possibly explain the plot functions in more detail instead of just using them.  We use the plot functions alot but I've noticed myself just copying the code from the notebooks instead of knowing how to use it myself.

**Comment by dealmeidavf** [comment]
I agree. Every function in the course should have a proper python docstring. It is on my to-do list.
